exercises/practice/resistor-color/resistor_color_test.gd

```diff
@@ -0,0 +1,23 @@
+func test_color_codes_black(solution_script):
+	return [false, solution_script.color_code("black")]
+
+func test_color_codes_white(solution_script):
+    return [false, solution_script.color_code('white"')]
```

Comment:
There is a small typo here, making us look for `white"` instead of `white`. After fixing it, I believe we should compare the result to 9 instead of false.

What worries me a bit is the fact that my test runner should detect if the type of variables is different (e.g. int vs bool) and fail in that case. But if your tests are passing, then something went wrong ...